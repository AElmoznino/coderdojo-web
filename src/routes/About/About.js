// @flow

import React from 'react'
import GridColumn from '../../components/Grid/GridColumn'
import H1 from '../../components/Typography/H1'

export const About = () => (
  <GridColumn>
    <H1>Om CoderDojo Webbskolan</H1>
    <p>
      Webbskolan är ett projekt för att lära barn och ungdomar att programmera
      för webben. Här finns ett urval av övningar i HTML, CSS och Javascript.
      Tanken är att man ska kunna jobba helt självgående.
    </p>
  </GridColumn>
)

export default About
