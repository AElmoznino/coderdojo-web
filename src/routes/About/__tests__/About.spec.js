import React from 'react'
import { shallow } from 'enzyme'
import About from '../About'

describe('components/About', () => {
  let component

  beforeEach(() => {
    component = shallow(<About />)
  })

  it('renders About', () => {
    expect(component).toMatchSnapshot()
  })
})
