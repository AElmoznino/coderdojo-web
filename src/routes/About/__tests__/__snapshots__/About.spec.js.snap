// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/About renders About 1`] = `
<styled.div>
  <styled.div>
    <styled.h1>
      Om CoderDojo Webbskolan
    </styled.h1>
    <p>
      Webbskolan är ett projekt för att lära barn och ungdomar att programmera för webben. Här finns ett urval av övningar i HTML, CSS och Javascript. Tanken är att man ska kunna jobba helt självgående.
    </p>
    <p>
      Sidan är optimerad för Google Chrome och anpassad för bärbar dator då det är vad barnen/ungdomarna har med sig till CoderDojo.
    </p>
  </styled.div>
</styled.div>
`;
